src/NuGet.Clients/NuGet.PackageManagement.UI/ViewModels/TitledPageViewModelBase.cs

```diff
@@ -0,0 +1,25 @@
+namespace NuGet.PackageManagement.UI.ViewModels
+{
+    public class TitledPageViewModelBase : ViewModelBase
+    {
+        private string _title;
+        public string Title
+        {
+            get => _title;
+            set
+            {
+                SetAndRaisePropertyChanged(ref _title, value);
+            }
```

Comment:
Nit: could be more concise

```suggestion
            set => SetAndRaisePropertyChanged(ref _title, value);
```